components/PageNavigation.vue

```diff
@@ -66,6 +66,17 @@
 
       &.nuxt-link-active {
         font-weight: bold;
+        &:after {
+          content: '';
+          position: absolute;
+          border-bottom: solid 3px #0771ce;
```

Comment:
Would you mind updating the colour of the border? You can use the variable $blue.